### PR TITLE
Remove deprecated Gecko macros

### PR DESCRIPTION
**Pull request by teoli2003**
This macro is deprecated, but we still had a few occurrences in outdated pages or comments.

This fixes it.

### CODE REVIEW

File: files/en-us/web/api/document/height/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ browser-compat: api.Document.height
 ---
 {{APIRef("DOM")}} {{deprecated_header}}{{Non-standard_header}}
 
-> **Note:** Starting in {{Gecko("6.0")}}, `document.height` is
+> **Note:** `document.height` is
```
Comment (wbamberg) [inline files/en-us/web/api/document/height/index.md:19]: Surely we can delete this page, and its friend below?

File: files/en-us/web/api/document/createentityreference/index.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ browser-compat: api.Document.createEntityReference
 ---
 {{ApiRef("DOM")}}{{deprecated_header}}
 
-Prior to {{ gecko("7.0") }} this method showed up as present, due to bug {{ bug("9850") }}, it always only returned null. The only workaround is to create a text node, CDATA section, attribute node value, etc. which has the value referred to by the entity, using [Unicode escape sequences](/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#unicode_escape_sequences) or [fromCharCode()](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCharCode) as necessary.
+Prior to Firefox 7.0 this method showed up as present, due to bug {{ bug("9850") }}, it always only returned null. The only workaround is to create a text node, CDATA section, attribute node value, etc. which has the value referred to by the entity, using [Unicode escape sequences](/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#unicode_escape_sequences) or [fromCharCode()](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCharCode) as necessary.
```
Comment (wbamberg) [inline files/en-us/web/api/document/createentityreference/index.md:15]: Delete this page?

File: files/en-us/web/api/selection/modify/index.md [hunk 3]
```diff
@@ -40,12 +40,12 @@ modify(alter, direction, granularity)
...
     `"documentboundary"`.
 
-> **Note:** Gecko does **not** implement
+> **Note:** Firefox does **not** implement
 > `"sentence"`, `"paragraph"`, `"sentenceboundary"`,
 > `"paragraphboundary"`, or `"documentboundary"`. WebKit and Blink
 > do.
 
-> **Note:** Starting in {{Gecko("5.0")}}, the `"word"`
+> **Note:** Starting in Firefox 5, the `"word"`
```
Comment (wbamberg) [inline files/en-us/web/api/selection/modify/index.md:48]: This should all be in BCD but not worth doing that now I think.

File: files/en-us/web/api/svglengthlist/index.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ The `SVGLengthList` defines a list of {{ domxref("SVGLength") }} objects.
 
 An `SVGLengthList` object can be designated as read only, which means that attempts to modify the object will result in an exception being thrown.
 
-> **Note:** Starting in {{Gecko("5.0") }},the `SVGLengthList` DOM interface is now indexable and can be accessed like arrays
+> **Note:** Tthe `SVGLengthList` DOM interface is indexable and can be accessed like an array.
```
Comment (wbamberg) [inline files/en-us/web/api/svglengthlist/index.md:21]:
```suggestion
An `SVGLengthList` is indexable and can be accessed like an array.
```

I don't think this should be a note.

File: files/en-us/web/api/svgnumberlist/index.md [hunk 5]
```diff
@@ -17,7 +17,7 @@ The `SVGNumberList` defines a list of {{ domxref("SVGNumber") }} objects.
 
 An `SVGNumberList` object can be designated as read only, which means that attempts to modify the object will result in an exception being thrown.
 
-> **Note:** Starting in {{ Gecko("5.0") }},the `SVGNumberList` DOM interface is now indexable and can be accessed like arrays.
+> **Note:** The `SVGNumberList` DOM interface is now indexable and can be accessed like an array.
```
Comment (wbamberg) [inline files/en-us/web/api/svgnumberlist/index.md:20]:
```suggestion
An `SVGNumberList` is indexable and can be accessed like an array.
```

File: files/en-us/web/api/svgtransformlist/index.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ The `SVGTransformList` defines a list of {{ domxref("SVGTransform") }} objects.
 
 An `SVGTransformList` object can be designated as read only, which means that attempts to modify the object will result in an exception being thrown.
 
-> **Note:** Starting in {{Gecko("9.0") }},the `SVGTransformList` DOM interface is now indexable and can be accessed like Arrays
+> **Note:** The `SVGTransformList` DOM interface is indexable and can be accessed like an array.
```
Comment (wbamberg) [inline files/en-us/web/api/svgtransformlist/index.md:20]:
```suggestion
An `SVGTransformList` is indexable and can be accessed like an array.
```

**Review by wbamberg — COMMENTED** [review]
Nice! The main suggestion is to delete pages on some features that haven't been supported for a very long time.

File: files/en-us/web/api/document/width/index.md [hunk 7]
```diff
@@ -16,7 +16,7 @@ browser-compat: api.Document.width
 ---
 {{APIRef("DOM")}} {{deprecated_header}}{{Non-standard_header}}
 
-> **Note:** Starting in {{Gecko("6.0")}}, `document.width` is
+> **Note:**  `document.width` is
```
Comment (wbamberg) [inline files/en-us/web/api/document/width/index.md:19]: Delete this page?

File: files/en-us/web/api/document/createentityreference/index.md (hunk 2 above)
Comment (teoli2003) [reply files/en-us/web/api/document/createentityreference/index.md:15]: Yes. Commit incoming

File: files/en-us/web/api/document/height/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/document/height/index.md]: Dibe

File: files/en-us/web/api/document/width/index.md (hunk 7 above)
Comment (teoli2003) [reply files/en-us/web/api/document/width/index.md]: Done

**Review by wbamberg — APPROVED** [review]
👍 nice!

**Pull request merged**